app/src/main/java/com/xmartlabs/gong/ui/Themes.kt

```diff
@@ -0,0 +1,208 @@
+package com.xmartlabs.gong.ui
+
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Typography
+import androidx.compose.material.darkColors
+import androidx.compose.material.lightColors
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.xmartlabs.gong.R
+
+object Themes {
+
+  @Composable
+  fun GongTheme(
+      darkTheme: Boolean = isSystemInDarkTheme(),
+      content: @Composable () -> Unit,
+  ) {
+    MaterialTheme(
+        colors = if (darkTheme) GongColors.darkThemeColors else GongColors.lightThemeColors,
+        typography = Typographies.gongTypography,
+        content = content,
+    )
+  }
+}
+
+object GongColors {
```

Comment:
What do you think about creating extensions for material `Colors` to add our own?

Reply:
you cannot do that, because you have to save it in your context. 
We have two options:
1. manage different colors (one for material and another for gong custom colors)
2. gong custom colors should contain material custom colors, so you can convert a gong color to a material color